main.go

```diff
@@ -453,7 +453,7 @@ func prepareConfig(parseOpts *cliOptions, conf *ffuf.Config) error {
 	}
 
 	// Handle copy as curl situation where POST method is implied by --data flag. If method is set to anything but GET, NOOP
-	if conf.Method == "GET" {
+	if conf.Method == "GET" && len(parseOpts.request) == 0 {
```

Comment:
Can you add comment here to point out why this condition exists. It'll help figuring out what's going on in the future :)